app/models/github_repository.rb

```diff
@@ -58,6 +116,12 @@ def present?(**options)
     self.class.present?(@client, @id, options)
   end
 
+  def classroom_configs?(**options)
```

Comment:
What does this method do? `repository.classroom_configs?` is a little confusing.